models/api/coursework-material/index.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require("mongoose");
 
-const {Schema} = mongoose;
-const {ObjectId} = Schema.Types;
+const { Schema } = mongoose;
+const { ObjectId } = Schema.Types;
```

Comment:
I'm good with this being a standard in the project. I just updated my Webstorm code style rules to do this when I beautify a file. (These were changed because of the old beautify settings in Webstorm)